spec/imsc-hrm.html

```diff
@@ -216,12 +216,18 @@ <h2>Documentation Conventions</h2>
 
   <section id='terms'>
     <h2>Terms and Definitions</h2>
-      <p><dfn>IMSC Document Instance</dfn>. A <dfn data-lt="Document Instance|Document Instances"
-      data-cite="ttml2#terms-document-instance">Document Instance</dfn> that conforms to any profile defined in any edition of
-      [[!IMSC]].</p>
-      
-      <p><dfn>empty ISD</dfn>. an <a>Intermediate Synchronic Document</a> with no <dfn
-      data-cite="IMSC#dfn-presented-region">presented region</dfn>.</p>
+    
+    <p><dfn>character</dfn>. The character code property of a <dfn data-cite="ttml2#terms-character-information-item">Character Information Item</dfn>.</p>
+
+    <p class="note">The term <a>character</a> is for practical purposes
+      the same as a [=code point=].</p>
```

Comment:
```suggestion
      the same as a <a>code point</a>.</p>
```

We need avoid mixing markdown and HTML.

Reply:
I didn't think that `[=...=]` _is_ markdown, but happy to change it to an `<a>` link regardless.

Reply:
Addressed in https://github.com/w3c/imsc-hrm/commit/ee04753e86c97b5930744c4351e5f440ebd0176e